ext/bcmath/tests/number/methods/round.phpt

```diff
@@ -19,11 +19,13 @@ foreach (RoundingMode::cases() as $mode) {
         '2.5',
         '-2.5',
     ] as $number) {
-        $func_ret = bcround($number, 0, $mode);
-        $method_ret = (new BcMath\Number($number))->round(0, $mode);
-        if ($method_ret->compare($func_ret) !== 0) {
-            echo "Result is incorrect.\n";
-            var_dump($number, $mode, $func_ret, $method_ret);
+        foreach ([0, 5, -5] as $scale) {
+            $func_ret = bcround($number, $scale, $mode);
+            $method_ret = (new BcMath\Number($number))->round($scale, $mode);
+            if ($method_ret->compare($func_ret) !== 0) {
+                echo "Result is incorrect.\n";
+                var_dump($number, $mode, $func_ret, $method_ret);
+            }
```

Comment:
The indentation added with foreach, making it difficult to see the change. I increased the test cases for scale 0 to test cases for scale 0, 5, and -5.